docs/changelog/85326.yaml

```diff
@@ -0,0 +1,24 @@
+pr: 85326
+summary: Deprecate LDAP/AD realm settings missing bind password for bind DN
+area: Authentication
+type: deprecation
+issues:
+ - 47191
+deprecation:
```

Comment:
My current understand is that this won't automatically create migration docs and will stay in the release notes instead. Please let me know if that's not the case.